deepinv/training_utils.py

```diff
@@ -527,3 +527,52 @@ def test(
         wandb.log({"Test PSNR": test_psnr}, step=step)
 
     return test_psnr, test_std_psnr, linear_psnr, linear_std_psnr
+
+
+def train_normalizing_flow(
```

Comment:
I would leave it here for now, it may be the most convenient. However we are heavily refactoring the training class atm and we would probably want to make this training function compatible with the (upcoming) Trainer. For now I would not change what you've done if this is convenient. Only thing is I would not load it by default in the global `deepinv/__init__.py` file since this is going to change.